src/actions/post_build.rs

```diff
@@ -76,16 +76,16 @@ impl PostBuildHandler {
                         t.unpark();
                     }
 
-                    self.notifier.notify_end();
```

Comment:
I think we can call `self.notifier.notify_begin_diagnostics()` `self.notifier.notify_end_diagnostics()` only here, in case of `BuildResult::Success`. Since the diagnosticsBegin/End changed meaning some time ago and only indicate indexing operation progress, I think it's fair only to do that when we have something meaningful to do.

Reply:
Ok. I fix.